src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/book';
+import categoryReducer from './categories/categories';
+
+const store = configureStore({
+  reducer: {
+    book: bookReducer,
+    category: categoryReducer,
+  },
```

Comment:
- [ ] As mentioned in the project requirements, Please use `combineReducers` to combine both reducers into a root reducer.

Reply:
You can check my code here : 


import { configureStore } from '@reduxjs/toolkit';
import { combineReducers } from 'redux';
import bookReducer from './books/book';
import categoryReducer from './categories/categories';

const rootReducer = combineReducers({
  bookReducer,
  categoryReducer,
});

const store = configureStore(rootReducer);
export default store;